app/views/content/subjects/physics/_article.html.erb

```diff
@@ -0,0 +1,168 @@
+<div class="row inset">
+<section class="col col-720">
+  <p>
+    Teaching physics is an exciting and rewarding career. Schools need more specialist physics teachers, so it's a great choice of subject.
+  </p>
+
+  <p>
+    If you train to teach physics, you could also be eligible receive a scholarship or bursary of up to £29,000.
+  </p>
+  </section>
+
+   <section class="col col-space-s col-content-max">
+    <%= render TextBlockWithTwoImagesComponent.new(
+      heading: "Spark curiosity in physics",
+      colour: "yellow",
+      image_1_path: "static/content/hero-images/0012.jpg",
+      image_1_alt: "Pupils listening to a science lesson.",
+      image_2_path: "static/content/hero-images/0007.jpg",
+      image_2_alt: "Teacher talking to a pupil in a classroom.",
+    ) do %>
+       <p>
+    As a physics teacher, you’ll be engaging students in a wide range of experiments that can spark their interest.
+  </p>
+
+  <p>
+    You can inspire your students to consider rewarding careers from tackling climate change to artificial intelligence.
```

Comment:
just wondering if there should be a bit of a bigger gap between these two paragraphs? (ignore me if already discussed)